src/targets/enemies/goblin.js

```diff
@@ -1,5 +1,6 @@
 import { Enemy } from "./enemy.js";
 import NormalBones from "../../items/bones/normal-bones.js";
+import AirTalisman from "../../items/tools/talismans/air-talisman.js";
```

Comment:
Was this supposed to be included previously?

Reply:
Yep lol